CHANGELOG-1.1.md

```diff
@@ -1,3 +1,8 @@
+> **Note:**
+>
+> * Starting from September 2, 2020, all release notes of TiDB Operator will be maintained in [pingcap/docs-tidb-operator](https://github.com/pingcap/docs-tidb-operator).
+> * You can read the release notes of all versions of TiDB Operator at [PingCAP Docs](https://docs.pingcap.com/tidb-in-kubernetes/stable/release-1.1.4).
```

Comment:
We can also create a file that contains all release notes for TiDB Operator, like https://docs.pingcap.com/tidb/stable/release-notes. Then we can use that link here (with the 1.1 anchor link). But this PR is also acceptable currently.

Reply:
Good idea. I'll create a file like that later.
Let's merge this PR for now.